students/piechowski.michal/lesson_03_functions/the_length_of_the_segment.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+from math import sqrt
+
+x1 = float(input())
+y1 = float(input())
+x2 = float(input())
+y2 = float(input())
+
+distance = sqrt((x2 - x1) ** 2 + (y2 - y1) ** 2)
```

Comment:
zdaje się, że to miała być funkcja `distance`, która liczy odległość między dwoma punktami a nie zmienna.